tests/end2end/test_hints_html.py

```diff
@@ -146,3 +151,51 @@ def test_word_hints_issue1393(quteproc, tmpdir):
         quteproc.wait_for(message='hints: *', category='hints')
         quteproc.send_cmd(':follow-hint {}'.format(hint))
         quteproc.wait_for_load_finished('data/{}'.format(target))
+
+
+@pytest.mark.parametrize('min_len', [0, 3])
```

Comment:
This isn't really the right place for a test like this (it should be in a new `test/unit/browser/test_hints.py`), but I'll move it while merging.

Reply:
Done in af638ec430be25c5f61c697a49efaeb7cade2bd6